rails_programming/mailers_advanced_topics/websockets_and_actioncable.md

```diff
@@ -8,12 +8,8 @@ There are often times where you want to keep that connection open so that the se
 Look through these now and then use them to test yourself after doing the assignment.
```

Comment:
Let's just drop this blurb for now because it doesn't line up with what we're wanting Learning Outcomes to be anymore. 

We will probably put something back here in the future, but at the moment I see a handful of different Learning Outcome section blurbs in the curriculum and it seems like we need to standardize on one in the future and apply it in all of the lessons.